clientapi/routing/routing.go

```diff
@@ -68,14 +68,17 @@ func Setup(
 				Code: http.StatusOK,
 				JSON: struct {
 					Versions []string `json:"versions"`
-				}{[]string{
+					UnstableFeatures map[string]bool `json:"unstable_features"`
+				}{Versions: []string{
 					"r0.0.1",
 					"r0.1.0",
 					"r0.2.0",
 					"r0.3.0",
 					"r0.4.0",
 					"r0.5.0",
 					"r0.6.1",
+				},UnstableFeatures: map[string]bool{
```

Comment:
It might make sense to only list MSCs here that are enabled in the config?

Reply:
Hi! Is there any variable which stores list of enabled MSCs. I am still understanding the codebase.
Thank You

Reply:
The `routing.Setup` function in `clientapi/routing/routing.go` has access to `cfg *config.ClientAPI`, which contains access to some of the relevant config for the client API.

However, there's no reference in that struct to the `MSCs` struct from `config_mscs.go`, so you may need to update `setup/config/config_clientapi.go` first so there's a reference to the `MSCs` struct.